### PR TITLE
Fix broken links on site pages - links fixed on the Clients page

### LINKED ISSUE #508: Clients page refers to incorrect URLs of our clients' website
Linked to parent issue #503.

Minor differences in the way the URL has been written cause the link to return a 404 or 503, especially if the `https` protocol is used.

### PR DESCRIPTION
**Pull request by neomatrix369**
Links to some of the clients' website on https://codurance.com/clients/ are incorrect, this PR fixes them.

Fixes #508.

### CODE REVIEW

File: clients/index.html [hunk 1]
```diff
@@ -163,7 +163,7 @@
       <div class="col-md-10">
         <h3>Crowdmix</h3>
         <ul class="list-inline">
-          <li><i class="fa fa-globe color-green"></i> <a class="linked" href="http://crowdmix.me">crowdmix.me</a></li>
+          <li><i class="fa fa-globe color-green"></i> <a class="linked" href="http://www.crowdmix.me">crowdmix.me</a></li>
```
Comment (cyplo) [inline clients/index.html]: this site does not support https nor does display correctly in chrome, do we want it mentioned ?

Comment (neomatrix369) [reply clients/index.html]: Good point, I wanted to bring this up in our discussions. I'm unsure about it, although we have mentioned them for some time on our site.

Comment (cyplo) [reply clients/index.html]: Would you like to make this pull request separate from that discussion ? I.e. removal of the link would be a separate PR ?

Comment (neomatrix369) [reply clients/index.html]: I'll undo changes for this site - as its broken irrespective. 👍 Thanks @cyplo

**Review by cyplo — APPROVED** [review]

**Comment by neomatrix369** [comment]
@cyplo Links changed to `https` where available

**Comment by neomatrix369** [comment]
@cyplo As agreed, all links are now using `https` protocol, except for one which does not support it.

**Comment by cyplo** [comment]
Looks good to me :) Would it be alright with you to check the links in #511 as well ?

**Comment by neomatrix369** [comment]
@cyplo applying this to #511 might be a different PR I think - the scope of #511 is to fix existing broken links in the blogs.

If  nothing else I needed to be done, can we merge this PR pls

**Pull request merged**